src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -166,7 +166,7 @@ public boolean isAACLC() {
 	/**
 	 * @return True if the audio codec is AAC HE.
 	 */
-	public boolean isAACHE() {
+	public boolean isHEAAC() {
```

Comment:
Sorry, I don't remember. Which PR? The important thing to me isn't what it's called but that it's consistent, so that it's easy to look up. My logic is that ```AAC``` is the "main category" and "HE" is the "variant", so when you want to look it up you'll naturally start by typing ```A```.